packages/components/src/modal/style.scss

```diff
@@ -121,13 +102,19 @@
 
 // Modal contents.
 .components-modal__content {
-	box-sizing: border-box;
-	height: 100%;
+	flex: 1;
+	border-radius: 0 0 $radius-block-ui $radius-block-ui;
+	margin-top: $header-height;
 	padding: 0 $grid-unit-40 $grid-unit-30;
+	overflow: auto;
 
-	// Rules inside this query are only run by Microsoft Edge.
-	// This is a companion top padding to the fixed rule in line 77.
-	@supports (-ms-ime-align:auto) {
```

Comment:
I'm so glad we can remove legacy hacks like these.

Reply:
🎉  Funny how the changes here should improve support in the browsers we don't need to worry about anymore.